### PR TITLE
Add remove edit

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
- Removed all hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task (add a new element to the array).
- Implemented a function for deleting a task (remove an element from the array).
- Implemented a function for editing task descriptions.
- All changes to the To-Do List should be saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import './style.css';
+
+const taskSection = document.querySelector('.tasks');
+const userTask = document.querySelector('.user-task');
+const tasksArray = JSON.parse(localStorage.getItem('task')) || [];
+let MOOD = 'CREATE';
+let tmp;
+class Task {
+  constructor(userTask) {
+    this.id = tasksArray.length;
```
Comment (ShoiraTa) [inline src/index.js]: - [ ] The id of the array should start from `1`, if a 5th task is added to the list, the index of that task should equal to `5`. In your case the array starts from `0` and 5th element would have id of `4`. Kindly make the id start from 1. You an simply do it by `this.id = tasksArray.length + 1` 😉

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+});
+
+displayTask(tasksArray);
+
+taskSection.addEventListener('click', (e) => {
+  if (e.target.classList.contains('span')) {
+    tasksArray.splice(e.target.parentElement, 1);
+    displayTask(tasksArray);
+    localStorage.setItem('task', JSON.stringify(tasksArray));
+  }
```
Comment (ShoiraTa) [inline src/index.js]: - [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. So if array has index `1,2,3,4,5,` after deleting 3rd element indexes should get updated and be:  `1,2,3,4`

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @TahenyBELGUITH ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- A function for adding a new task works properly ✔️ 
- A function for editing task descriptions  works properly ✔️ 
- A function for deleting a task  works properly ✔️ 
- Professional documentation, Pull Request and commits history! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @TahenyBELGUITH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**